include/baresip.h

```diff
@@ -1426,6 +1433,7 @@ int  stream_start_mediaenc(struct stream *strm);
 int  stream_start_rtcp(const struct stream *strm);
 int  stream_enable(struct stream *strm, bool enable);
 int  stream_enable_tx(struct stream *strm, bool enable);
+int  stream_enable_rx(struct stream *strm, bool enable);
```

Comment:
this function can return "void" ?

Reply:
Compared with `stream_enable_tx`. It is in the API and there are cases where an error makes sense.
Both `stream_enable_tx()` and `stream_enable_rx()` doesn't evaluate the returned error code so far. --> Could be another PR, but is not related to this one.